storage/pkg/flagset/gateway.go

```diff
@@ -97,6 +97,13 @@ func GatewayWithConfig(cfg *config.Config) []cli.Flag {
 			EnvVars:     []string{"STORAGE_GATEWAY_DISABLE_HOME_CREATION_ON_LOGIN"},
 			Destination: &cfg.Reva.Gateway.DisableHomeCreationOnLogin,
 		},
+		&cli.StringFlag{
+			Name:        "storage-home-mapping",
+			Value:       "",
+			Usage:       "mapping template for user home paths to user-specific mount points",
```

Comment:
can you add an example here and in the changelog on how to use it. here an `eg. /home/{{...}}` is good enough.